Vagrantfile

```diff
@@ -7,18 +7,25 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.ssh.username = "vagrant"
   config.ssh.forward_agent = true
   config.vm.provision :shell, :path => "vagrant_setup.sh", :privileged => false
-  # Sync a folder between the host and all guests.
-  # Uncomment this line (and adjust as you like)
-  #config.vm.synced_folder "~/code", "/code"
 
   config.vm.define "pwn", primary: true do |u64|
-    u64.vm.box = "geerlingguy/ubuntu1604"
-    u64.vm.network "private_network", ip: "10.10.10.10"
-    u64.vm.provider "virtualbox" do |vb|
-      vb.name = "pwn ubuntu"
+    u64.vm.provider "virtualbox" do |vb, override|
+      override.vm.box ="geerlingguy/ubuntu1604"
+      override.vm.network "private_network", ip: "10.10.10.10"
+      # Sync a folder between the host and all guests.
```

Comment:
I agree, I did this because libvirt's file sharing kind of sucks and relies on NFS, which requires prior setup on the host. I was planning to try and research this and see if I could streamline the system somehow for future releases, but right now libvirt users will have to manage folder sharing outside of vagrant (using SMB or something similar). 

I was going to add a note about this to the Vagrantfile but I didn't want to bloat the project with complicated instructions and gotcha's for libvirt. Think we should add it anyway so people aren't confused?